app/src/androidTest/java/com/github/se/travelpouch/MainActivityTest.kt

```diff
@@ -13,9 +11,5 @@ class MainActivityTest {
 
   @get:Rule val composeTestRule = createAndroidComposeRule<MainActivity>()
 
-  @Test
-  fun testMainActivityDisplaysCorrectContent() {
-    // Assert that the Surface with the "MainScreenContainer" test tag is displayed
-    composeTestRule.onNodeWithTag("MainScreenContainer").assertIsDisplayed()
-  }
+  @Test fun testMainActivityDisplaysCorrectContent() {}
```

Comment:
I think that this test and the whole file by the way should be removed it is empty.

Reply:
Done. Thank you, I'd forgotten!